chaincfg/params.go

```diff
@@ -50,6 +50,16 @@ type Checkpoint struct {
 	Hash   *chainhash.Hash
 }
 
+// DNSSeed identifies a DNS seed.
+type DNSSeed struct {
+	// Host defines the hostname of the seed.
+	Host string
+
+	// HasFiltering defines whether the seed supports filtering
+	// by wire.ServiceFlag's.
```

Comment:
nit: I don't think there is any reason for this to possessive.  Perhaps just `filtering by service flags (wire.ServiceFlag).`